default.conf

```
@@ -16,9 +19,23 @@ server {
         return 404;
     }
 
+    location ~* \.(html)$ {
+        add_header Cache-Control "public, max-age=0, must-revalidate";
+    }
+
+     location /js/bot/media {
```

```suggestion
     location /media {
```
We've moved media to the root folder in the new builds

---

Thanks Bala. Just found it out in the latest image.Will make the change.